.github/workflows/main.yml

```diff
@@ -217,6 +217,7 @@ jobs:
             python: '3.6'
           # Test alternate .NETs
           - java: '8'
+            # Current release info can be found at https://dotnet.microsoft.com/download/dotnet/5.0
```

Comment:
Not strictly related to this PR, but this can be helpful in the future. This is kinda trivial, too. This got added as I was trying to find a way to reliably cause an EOF to happen on Windows in CI/CD. Spoiler: I did not succeed, for some reason.